ldasw/.htaccess

```diff
@@ -17,5 +17,8 @@ RewriteCond %{HTTP_ACCEPT} text/\* [OR]
 RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
 RewriteCond %{HTTP_ACCEPT} ^.*application/rdf\+xml.* 
 RewriteRule ^university$ https://raw.githubusercontent.com/PaavoNykanen/Ontologies/main/LinkedData/UniversityOntology/university.owl [R=303,NE,L]
+RewriteRule ^university$ https://raw.githubusercontent.com/PaavoNykanen/Ontologies/main/LinkedData/UniversityOntology/university.rdf [R=303,NE,L]
```

Comment:
This one is not reachable, as the previous one will trigger the rule

Reply:
Do you have a suggestion how to fix this? Maybe adding 'RewriteCond %{HTTP_ACCEPT} ^.*application/owl\+xml.* ' before the owl link and the the rdf condition for the rdf file?

Or should the owl or rdf be moved to the end as a default response?

Reply:
you have to choose on which file to return when you are requested application/rdf+xml

Reply:
So if I left the ref+xml condition for the .rdf file, is there a similar condition for owl? Like application/owl+xml?

Reply:
OWL is not a serialization, right? I would leave only the rdf file. Or if you want to return an .owl file when people ask for rdf, then leave just the OWL file. It's up to you to decide what is being returned.

Reply:
well, `.rdf` is not a serialization either :)